pkg/kubelet/config/config.go

```diff
@@ -167,6 +167,10 @@ func (s *podStorage) Merge(source string, change interface{}) error {
 		if len(updates.Pods) > 0 {
 			s.updates <- *updates
 		}
+		// Only add reconcile support here, because kubelet doesn't support Snapshot update now.
+		if len(reconciles.Pods) > 0 {
+			s.updates <- *reconciles
+		}
```

Comment:
I think each pod should only belong to one set (adds, updates, deletes, or reconciles), so the orders should not matter. We don't guarantee the order across pods anyway.